connectors/src/connectors/intercom/index.ts

```diff
@@ -339,6 +339,12 @@ export async function fullResyncIntercomSyncWorkflow(
   const toBeSignaledHelpCenterIds = helpCentersIds.map((hc) => hc.helpCenterId);
   const toBeSignaledTeamIds = teamsIds.map((team) => team.teamId);
 
+  // We reset the lastUpsertedTs of all Help Center articles to make sure they are resynced
+  await IntercomArticle.update(
```

Comment:
I think it would be marginally better to instead pass an option to the sync worflow (fullResync: true) that skips the condition ?